cloud2cloud-connector/swagger.yaml

```diff
@@ -51,141 +51,144 @@ paths:
       parameters:
-        - $ref: '#/components/parameters/cloudId'
+        - $ref: "#/components/parameters/cloudId"
       responses:
         307:
           description: redirect to get authorization code
       security:
-      - c2c_connector_auth:
-        - empty
+        - c2c_connector_auth:
+            - empty
   /api/v1/clouds/{cloudId}/accounts/{accountId}:
     delete:
       tags:
         - Accounts
       summary: Delete linked account
       parameters:
-        - $ref: '#/components/parameters/cloudId'
-        - $ref: '#/components/parameters/accountId'
+        - $ref: "#/components/parameters/cloudId"
+        - $ref: "#/components/parameters/accountId"
       responses:
         200:
           description: successful operation
       security:
-      - c2c_connector_auth:
-        - empty
+        - c2c_connector_auth:
+            - empty
   /api/v1/oauthCallback:
     get:
       tags:
         - Accounts
       summary: Used for get authorization code for linked account
       parameters:
-        - $ref: '#/components/parameters/oauthCallback'
+        - $ref: "#/components/parameters/oauthCallback"
       responses:
         200:
           description: redirect to get autorization code
   /api/v1/events:
-    $ref: 'https://raw.githubusercontent.com/openconnectivityfoundation/core-extensions/ocfcloud-openapi/swagger2.0/oic.r.cloudopenapi.swagger.json#/paths/%2F%7BeventsUrl%7D'
+    $ref: "https://raw.githubusercontent.com/openconnectivityfoundation/core-extensions/ocfcloud-openapi/swagger2.0/oic.r.cloudopenapi.swagger.json#/paths/%2F%7BeventsUrl%7D"
 components:
   schemas:
     LinkedClouds:
       type: object
       properties:
         cloud:
-          $ref: '#/components/schemas/LinkedCloud'
+          $ref: "#/components/schemas/LinkedCloud"
     LinkedCloud:
       type: object
       properties:
-        Id:
+        id:
           type: string
           format: uuid
-        Name:
+        name:
           type: string
-        OAuth:
+        oauth:
           type: object
           properties:
-            ClientId:
+            clientID:
               type: string
-            ClientSecret:
+            clientSecret:
               type: string
-            Scopes:
+            scopes:
               type: array
               items:
                 type: string
-            Endpoint:
-              type: object
-              properties:
-                AuthUrl:
-                  type: string
-                TokenUrl:
-                  type: string
-                AuthStyle:
-                  type: string
-                  enum: [ AutoDetect, InParams, InHeader ]
-            Audience:
+            authURL:
               type: string
-            RedirectUrl:
+            tokenURL:
               type: string
-            AccessType:
+            authStyle:
               type: string
-            ResponseType:
+              enum: [autoDetect, inParams, inHeader]
```

Comment:
@jkralik @ondrejtomcik what's the convention for enum values? I've used camel case here but on lines 162-165 the subscription types are snake case. So it's inconsistent.

Reply:
HTTP API should be always camelCase.

Reply:
Hmm the event type names are specified in `OCF Cloud API for Cloud Services Specification` with snake_case names, so I'm not gonna touch those.